workflows/cloudify_system_workflows/snapshots/postgres.py

```diff
@@ -512,3 +518,13 @@ def dump_license_to_file(self, tmp_dir):
     def restore_license_from_dump(self, tmp_dir):
         dump_file = os.path.join(tmp_dir, LICENSE_DUMP_FILE)
         self._restore_dump(dump_file, self._db_name, table='licenses')
+
+    @staticmethod
+    def _print_postgres_config(config):
+        config_password = config.postgresql_password
+        config_username = config.postgresql_username
+        config.postgresql_password = '********'
+        config.postgresql_username = '********'
```

Comment:
Can be changed to:
`config.postgresql_password = config.postgresql_username = '********'`